src/notation/internal/notationinteraction.cpp

```diff
@@ -2218,6 +2218,25 @@ void NotationInteraction::addText(TextType type)
     notifyAboutSelectionChanged();
 }
 
+void NotationInteraction::toggleNavigator()
```

Comment:
I think need a better name, this name a little confusing. Seems, what this method shows and hides the navigator.

Reply:
renamed to toggleNavigatorOrientation

Reply:
renamed to toggleNavigator
now method shows and hides the navigator